docs/mining/proof-of-work/pool-mining/gominer/linux.md

```diff
@@ -0,0 +1,99 @@
+
+### **Set Up the *gominer* Configuration File**
+
+> Step 1: Choose Mining Pool
+
+Before we begin this step, it is important that you've already signed up for a mining pool account (if required by your chosen mining pool, you'll have to create a worker at the pool's website). Record your worker's login and password, and the pool's stratum+tcp address:port for later use.
+
+If this is new to you, here are a few guides for creating workers on some of the Decred mining pools (WARNING: do not forget your PIN you set during account creation - you will be unable to withdraw any mined DCR without it):
+
+- [https://dcr.maxminers.net/index.php?page=gettingstarted](https://dcr.maxminers.net/index.php?page=gettingstarted)
+- [https://dcr.suprnova.cc/index.php?page=gettingstarted](https://dcr.suprnova.cc/index.php?page=gettingstarted)
+- [https://www2.coinmine.pl/dcr/index.php?page=gettingstarted](https://www2.coinmine.pl/dcr/index.php?page=gettingstarted)
+- [https://pool.mn/dcr/index.php?page=gettingstarted](https://pool.mn/dcr/index.php?page=gettingstarted)
+
+> Step 2: Verify Device ID#
+
+Before setting up the configuration file, it is vital you find out which devices, as identified by gominer, you wish to use to mine (especially if you want to avoid mining with your CPU or IGP). To do this, open a terminal window, navigate to the location you installed gominer above (e.g. `~/decred/gominer-linux-amd64-cuda-v1.0.0`), type `./gominer -l` or `./rungominer.sh -l` if you're using the CUDA version, press Enter, and a list of the devices you can mine with will be displayed. Record which device IDs you would like to use. Hint: Your video cards may be listed by their codename.
+
+> Step 3: Edit Configuration File
+
+Open the `sample-gominer.conf` file with your text editor of choice. For each setting we are about to change, simple delete the semi-colon before the setting variable after the `=` sign. 
+
+Within the "General settings" section:
+
+- set `devices=` to the Device ID#(s) recorded by using the `./gominer -l` or `./rungominer.sh -l` command in Step 2. Multiple devices should be separated by comma, e.g. `devices=2,3`.
+
+Within the "Mining settings" section:
+
+- set `pool=` to the stratum+tcp address:port of the mining pool of your choice, e.g. decredpool.org uses `stratum+tcp://stratum.decredpool.org:3333` (WARNING: the http://address:port will not work with gominer - stratum+tcp *must* be used).
+- set `pooluser=` to your worker's login.
+- set `poolpass=` to your worker's password.
+
+After these changes have been made, save the file.  Make a copy of it and call it gominer.conf in the gominer config directory:
+
+```no-highlight
```

Comment:
I would start this with
```
mkdir -p ~/.gominer
```
then you can get rid of the warning about it failing below.